config/locales/client.en.yml

```diff
@@ -5221,7 +5221,16 @@ en:
         reports:
           trend_title: "%{percent} change. Currently %{current}, was %{prev} in previous period."
           percent_change_tooltip: "%{percent} change."
-          percent_change_tooltip_previous_value: "Was %{previousValue} %{previousPeriod}."
+          percent_change_tooltip_previous_value:
+            yesterday:
+              one: "Was %{count} two days ago."
```

Comment:
Why nor saying "yesterday" then?

Reply:
It's a bit incongruous, but it said "two days ago" before so I don't want to change the original intent.

Reply:
Means the original number. Like "was 30 % two days ago".